packages/app/src/cli/models/extensions/specifications/function.ts

```diff
@@ -73,6 +73,7 @@ const spec = createExtensionSpecification({
     'fulfillment_constraints',
     'order_routing_location_rule',
     'local_pickup_delivery_option_generator',
+    'delivery_options_reducer',
```

Comment:
This list is a legacy list for extensions that had in the toml: `type: {the_api_type}` like `type: ´fulfillment_constraints``

If this is a new api type, all new functions using this will have `type: function` right? so this shouldn't be necessary?